ecs-image-build/docker_start.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+#
+# Start script for certificates-orders-api
+
+PORT=8080
+
+exec java -jar -Dserver.port="${PORT}" "certificates-orders-api.jar"
```

Comment:
Think this might be the full name
i.e. certificates.orders.api.ch.gov.uk.jar
better double check